policyengine_us/tests/policy/baseline/gov/states/ct/tax/income/credits/ct_property_tax_credit.yaml

```diff
@@ -0,0 +1,49 @@
+- name: Single household with $49,000  agi and  $100 property tax 
+  period: 2022
+  input:
+    state_code: CT
+    ct_agi: 49_000
+    filing_status: SINGLE
+    ct_property_tax: 100
+  output:
+    ct_property_tax_credit: 100
+
+- name: Separate household with $75,000  agi and  $250 property tax 
+  period: 2022
+  input:
+    state_code: CT
+    ct_agi: 75_000
+    filing_status: SEPARATE
+    ct_property_tax: 250
+  output:
+    ct_property_tax_credit: 0
+
+- name: Separate household with $45,000  agi and  $250 property tax
+  period: 2022
+  input:
+    state_code: CT
+    ct_agi: 45_000
+    filing_status: SEPARATE
+    ct_property_tax: 250
+  output:
+    ct_property_tax_credit: 175
+
+- name: Head of household with $60,000  agi and  $400 property tax 
+  period: 2022
+  input:
+    state_code: CT
+    ct_agi: 60_000
+    filing_status: HEAD_OF_HOUSEHOLD
+    ct_property_tax: 400
+  output:
+    ct_property_tax_credit: 255
+
+- name: Joint household with $130,500  agi and  $500 property tax
+  period: 2022
+  input:
+    state_code: CT
```

Comment:
need to include: ct_property_tax_credit_eligible: true instead of state_code: CT